llarp/service/outbound_context.cpp

```diff
@@ -317,16 +321,30 @@ namespace llarp
     {
       if(markedBad)
         return false;
-      if(path::Builder::ShouldBuildMore(now))
-        return true;
-      return !ReadyToSend();
+      const bool should =
```

Comment:
consider renaming variable `should` to something more descriptive.  Context gives decent clarity here, but it can't hurt to be more explicit.